src/main/java/screens/InfoScreen.java

```diff
@@ -0,0 +1,19 @@
+package screens;
+
+import javafx.scene.control.Label;
+import javafx.scene.layout.GridPane;
+
+/**
+ * Implementation of the window that handles showing information of the graph.
+ */
+public class InfoScreen extends GridPane {
+
+    /**
+     * Constructor.
+     */
+    public InfoScreen() {
+        getStyleClass().addAll("pane");
```

Comment:
This does not actually display any information yet. Should that be in here or is that going to be added later?